src/Log/LogService.php

```diff
@@ -417,10 +425,21 @@ public function getDefaultFormat()
             $this->defaultFormat = $format;
         }
 
-        $runId = $this->getRunId();
+        $format = $this->addRunIdToFormat($this->defaultFormat);
 
-        $format = rtrim(preg_replace('/%runId%/', $runId, $this->defaultFormat), PHP_EOL) . PHP_EOL;
+        return $format;
+    }
 
+    /**
+     * Add RunId to format string.
+     *
+     * @param $format string
+     * @returns string
+     */
+    public function addRunIdToFormat($format)
+    {
+        $runId = $this->getRunId();
+        $format = rtrim(preg_replace('/%runId%/', $runId, $format), PHP_EOL) . PHP_EOL;
```

Comment:
Try not to modify variables from function arguments ($format). In some languages and styles that is frowned upon. In Java these arguments can be declared *final*. It would be okay here, but should be avoided.